test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -104,28 +111,49 @@ public static void afterClass()
     @Before
     public void beforeEach()
     {
-        cluster.schemaChange("DROP KEYSPACE IF EXISTS harry");
-        cluster.schemaChange("CREATE KEYSPACE harry WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};");
+        cluster.schemaChange("DROP KEYSPACE IF EXISTS " + KEYSPACE);
+        cluster.schemaChange("CREATE KEYSPACE " + KEYSPACE + " WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};");
     }
 
     @Test
     public void simplifiedSaiTest()
     {
-        withRandom(rng -> basicSaiTest(rng, SchemaGenerators.trivialSchema("harry", "simplified", 1000).generate(rng)));
+        withRandom(rng -> saiTest(rng,
```

Comment:
asked this in slack, why `withRandom`?  this tests a single seed, but that is very slow to find issues; we should run multiple seeds each time we run.

i posted one failing example below... when the schema has < 3 columns and all columns are `boolean` the new partition unique logic will infinite loop... you won't see this for over a year given we run so few seeds per CI run.